idd/Energy+.idd.in

```diff
@@ -64816,50 +65000,43 @@ WaterHeater:HeatPump,
        \object-list ZoneNames
        \note Used only if Inlet Air Configuration is ZoneAirOnly or ZoneAndOutdoorAir.
        \note Otherwise, leave field blank.
-  A14, \field Tank Object Type
```

Comment:
@nmerket Object Type fields should remain.  Curious what the motivation was to remove these?  
@Myoldmopar How can we document this intent to avoid future redos?